contents/tutorials/framer-analytics.md

```diff
@@ -0,0 +1,112 @@
+---
+title: How to set up Framer analytics, session replay, and more
```

Comment:
Should we call it session recording? I think this is more likely to be searched for than session replay

```suggestion
title: How to set up Framer analytics, session recording, and more
```

Reply:
I think they are about the same, specifically for Framer, it doesn't seem like people are searching either yet, so I'll just keep replay to stay internally consistent.